lib/tasks/taxonomy/fixup_taxon_urls.rake

```diff
@@ -0,0 +1,37 @@
+namespace :taxonomy do
+  desc "fixup taxon URLs"
+  task fixup_taxon_urls: :environment do
+    total = RemoteTaxons.new.search.search_response["total"]
+    taxons = RemoteTaxons.new.search(per_page: total).taxons
```

Comment:
Would it be more efficient to use

```
      Services.publishing_api.put_content(content_id, payload)
      Services.publishing_api.publish(content_id, "minor")
```

instead of `Taxonomy::PublishTaxon`?